GiniVision/Classes/Core/Extensions/UIColor.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIColor.swift
+//  GiniVision
+//
+//  Created by Nadya Karaban on 12.10.20.
+//
+
+import Foundation
+
+extension UIColor {
+    func colorFromGiniColor(giniColor: GiniColor) -> UIColor {
```

Comment:
We could make this static and also rename it to better adhere to the swift method naming style:
```suggestion
    static func from(giniColor: GiniColor) -> UIColor {
```
By making it static we don't have to create a throwaway UIColor instance just to call this method.

Reply:
What do you think of renaming it to `from(giniColor:)`? Usually having only the parameter name is enough, like CGRect's [`frame(withWidth: ...)`](https://developer.apple.com/documentation/coregraphics/cgrect/2903485-frame)